src/openrct2/rct12/RCT12.h

```diff
@@ -32,6 +32,15 @@ namespace OpenRCT2
 }
 namespace OpenRCT2::RCT12
 {
+    enum class ClimateType : uint8_t
+    {
+        CoolAndWet,
+        Warm,
+        HotAndDry,
+        Cold,
+        Count
```

Comment:
lowerCamelCase.

Reply:
It's just a copy of the existing type. I'll change it, though.